src/banking_stage.rs

```diff
@@ -52,19 +52,26 @@ impl BankingStage {
     ) -> (Self, Receiver<Vec<Entry>>) {
         let (entry_sender, entry_receiver) = channel();
         let shared_verified_receiver = Arc::new(Mutex::new(verified_receiver));
-        let poh_recorder = PohRecorder::new(bank.tick_height(), *last_entry_id, max_tick_height);
+        let working_leader = WorkingLeader {
```

Comment:
Why `WorkingLeader` and not `WorkingBank`?  Poh doesn't need to know anything about leaders.